Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.7-alpine
```

The current python release is 3.8.
```suggestion
FROM python:alpine
```